src/main/java/com/ericsson/ei/subscriptionhandler/SubscriptionValidator.java

```diff
@@ -129,7 +131,9 @@ private void validateNotificationMeta(String notificationMeta) throws Subscripti
     private void validateNotificationType(String notificationType) throws SubscriptionValidationException {
         String regexMail = "[\\s]*MAIL[\\\\s]*";
         String regexRestPost = "[\\s]*REST_POST[\\\\s]*";
-        if (!(Pattern.matches(regexMail, notificationType) || Pattern.matches(regexRestPost, notificationType))) {
+        if (notificationType == null) {
+            throw new SubscriptionValidationException("Required field NotificationType is missed");
```

Comment:
"is missing" or "Required field NotificationType has not been set."

Reply:
I will update and add check too every field. I forgot about it earlier.